py-core/app/util/scripts/start_node_snapshot.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Instead of using Bash, can we use the Docker SDK for Python along with other relevant Python libraries.

Reply:
Also, could you check whether we can replace these scripts with straightforward calls to the Docker Compose SDK?

I wonder whether things work like this:
- Using Docker Compose, Tezos, Conseil and Arronax all run within the same container.
- To talk to each other, they simply use the local ports, i.e. they talk purely within the same container.
- Nginx is used to expose the three application ports to the outside world, with the port numbers being set either by string replacing within a template dockerfile or some other cleverer means.